ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -70,3 +71,43 @@ def test_torch_cat(
         dim=unique_idx,
         out=None,
     )
+
+    
+# reshape
+@given(
+    dtype_value_shape=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        ),
+        ret_shape=True,
+    ),
+    copy=st.booleans(),
```

Comment:
Please remove the usage of the `copy` argument from the `test_torch_reshape` function too.